Test/GetMinuteQuarterTest.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+use PHPUnit\Framework\TestCase;
+
+require_once "./src/MyClass.php";
+class GetMinuteQuarterTest extends TestCase
+{
+    public $obj;
+
+    
+    /**
+     * @dataProvider positiveDataProvider
+     */
+    public function testPositive($minute, $expected)
+    {
+        $this->obj = new MyClass();
+        $this->assertEquals($expected, $this->obj->getMinuteQuarter($minute));
+    }
+
+    public function testNegative()
+    {
+        $this->obj = new MyClass();
+
+        $this->expectException(InvalidArgumentException::class);
+
+        $this->obj->getMinuteQuarter(75);
+    }
+
+    public function positiveDataProvider()
```

Comment:
Use type hints here as well because you use it in `MyClass`, you should be consistent